charts/certbot/templates/role.yaml

```diff
@@ -42,3 +42,14 @@ rules:
       - routes/custom-host
     verbs:
       - create
+  - apiGroups:
+      - networking.k8s.io
+    resources:
+      - networkpolicies
+    verbs:
+      - list
+      - get
+      - create
+      - update
+      - patch
+      - delete
```

Comment:
Please add a newline at the end of the file if you can thanks.

Reply:
done